<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AscAnalyseForm</class>
 <widget class="QWidget" name="AscAnalyseForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,5" rowminimumheight="0,0">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Can数据偏移:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_signalDataOffset">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>CanID偏移：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_signalIDOffset">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_OffsetByVersion">
         <property name="text">
          <string>按版本自动检测偏移位置</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>信号ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_signalID">
         <property name="text">
          <string>29F</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>偏移：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_signalOffset">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>长度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_signalLen">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>过滤方向：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_Tx">
         <property name="text">
          <string>Tx</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_Rx">
         <property name="text">
          <string>Rx</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="pushButton_AddSignal">
         <property name="text">
          <string>添加监测</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_DelSignal">
         <property name="text">
          <string>删除检测选项</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Asc:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string>Please Open Asc FIle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_UpdateFile">
           <property name="text">
            <string>更新文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_UpdateSignal">
           <property name="text">
            <string>过滤信号</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCustomPlot" name="plot" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="statusBar">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_UpdateFile</sender>
   <signal>clicked()</signal>
   <receiver>AscAnalyseForm</receiver>
   <slot>pushButton_UpdateFile_OnCliecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>958</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_AddSignal</sender>
   <signal>clicked()</signal>
   <receiver>AscAnalyseForm</receiver>
   <slot>pushButton_AddSignal_OnCliecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_DelSignal</sender>
   <signal>clicked()</signal>
   <receiver>AscAnalyseForm</receiver>
   <slot>pushButton_DelSignal_OnCliecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_UpdateSignal</sender>
   <signal>clicked()</signal>
   <receiver>AscAnalyseForm</receiver>
   <slot>pushButton_UpdateSignal_OnCliecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>960</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButton_UpdateFile_OnCliecked()</slot>
  <slot>pushButton_AddSignal_OnCliecked()</slot>
  <slot>pushButton_DelSignal_OnCliecked()</slot>
  <slot>pushButton_UpdateSignal_OnCliecked()</slot>
 </slots>
</ui>
